ExtraEnemyCustomization/Customizations/Models/BoneCustom.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Enemies;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+
+namespace EECustom.Customizations.Models
+{
+    public class BoneCustom : EnemyCustomBase, IEnemySpawnedEvent
+    {
+        public BoneTransform[] Bones { get; set; } = new BoneTransform[0];
+
+        public override string GetProcessName()
+        {
+            return "Bone";
+        }
+
+        public void OnSpawned(EnemyAgent agent)
+        {
+            foreach (var boneTransform in Bones)
+            {
+                TryApplyBoneTransform(agent, boneTransform);
+            }
+        }
+
+        private void TryApplyBoneTransform(EnemyAgent agent, BoneTransform boneTransform)
+        {
+            try
+            {
+                var transform = agent.Anim.GetBoneTransform(boneTransform.Bone);
+                transform.localPosition += boneTransform.Offset;
+                transform.localScale = boneTransform.Scale;
+                transform.localRotation.SetEulerAngles(transform.localEulerAngles + boneTransform.RotationOffset);
+                LogVerbose($"Applied Bone Setting: {transform.name}, offset: {boneTransform.Offset.ToString()}, scale: {boneTransform.Scale.ToString()} rotationOffset: {boneTransform.RotationOffset.ToString()}");
+            }
+            catch(Exception e)
+            {
+                LogError($"Bone Transform [{boneTransform.Bone}] were not set! : {e}");
```

Comment:
Maybe some more verbose logging? Like adding the agent game object's name to the error log